src/patternfly/components/Wizard/wizard-toggle.hbs

```diff
@@ -0,0 +1,9 @@
+<button class="pf-c-wizard__toggle{{#if wizard-toggle--modifier}} {{wizard-toggle--modifier}}{{/if}}"
+  aria-expanded="{{#if wizard--IsExpanded}}true{{else}}false{{/if}}"
+  aria-controls="{{id}}-nav"
+  aria-haspopup="true"
```

Comment:
I'm finding that this attribute in JAWS is really weird when you use it outside the context of a form. For example, if I'm navigating a form in JAWS, the components we have with `aria-haspopup` work great. But if you encounter this component outside of a form, it's behavior is a little janky. 

I'm hoping that what we have labeled as "Basic Dropdown" (without aria-haspopup, and implemented with similar keyboard interaction of a popover) will work for these use cases of Dropdowns and Selects that are not embedded in a form, but I haven't been able to test a POC of that to know for sure. For now, I'd suggest just keeping `aria-expanded` and removing `aria-haspopup`.

Reply:
Aaaaand I've been commenting on all the PRs that I think we can remove `aria-controls` from cases like this. I think we only need it for things like Tabs or buttons in a toolbar, where the controlled element is not immediately after the toggle (or focus isn't automatically shifted like in a modal or popover). Currently this attribute only seems to benefit JAWS users, where JAWS provides a way to navigate to the controlled element.